easybuild/easyconfigs/c/C-Reduce/C-Reduce-2.8.0-GCC-7.3.0-2.30.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'CMakeMake'
+
+name = 'C-Reduce'
+version = '2.8.0'
+
+homepage = 'http://embed.cs.utah.edu/creduce/'
+description = """C-Reduce is a tool that takes a large C, C++, or OpenCL
+file that has a property of interest (such as triggering a compiler bug)
+and automatically produces a much smaller C/C++ file that has the same
+property. It is intended for use by people who discover and report bugs
+in compilers and other tools that process source code. C-Reduce is
+released under a BSD-like license."""
+
+toolchain = {'name': 'GCC', 'version': '7.3.0-2.30'}
+
+source_urls = ['https://github.com/csmith-project/creduce/archive']
+sources = ['creduce-%(version)s.tar.gz']
+checksums = ['ce8f2ec77569073cc93191ac4606894a8731e8c118abb197566a12d8ecd65954']
+
+builddependencies = [
+    ('CMake', '3.12.1'),
+    ('flex', '2.6.4'),
+]
+
+dependencies = [
+    ('zlib', '1.2.11'),
+    ('Perl', '5.28.0'),
+    ('Clang', '6.0.1'),
+    ('Exporter-Lite', '0.08', '-Perl-5.28.0'),
```

Comment:
Use `%(perlver)s`?

Reply:
Doesn't work. EB crashes if i do. @boegel ??

Reply:
```python
('Exporter-Lite', '0.08', '-Perl-%(perlver)s')
```

Should work?

Reply:
It doesn't:
```
...
  File "/pfs/software/eb/amd64_ubuntu1604_bdw/software/Core/EasyBuild/3.8.1/lib/python2.7/site-packages/easybuild_framework-3.8.1-py2.7.egg/easybuild/framework/easyconfig/easyconfig.py", line 1475, in process_easyconfig
    ec = EasyConfig(spec, build_specs=build_specs, validate=validate, hidden=hidden)
  File "/pfs/software/eb/amd64_ubuntu1604_bdw/software/Core/EasyBuild/3.8.1/lib/python2.7/site-packages/easybuild_framework-3.8.1-py2.7.egg/easybuild/framework/easyconfig/easyconfig.py", line 389, in __init__
    self.parse()
  File "/pfs/software/eb/amd64_ubuntu1604_bdw/software/Core/EasyBuild/3.8.1/lib/python2.7/site-packages/easybuild_framework-3.8.1-py2.7.egg/easybuild/framework/easyconfig/easyconfig.py", line 553, in parse
    self.generate_template_values()
  File "/pfs/software/eb/amd64_ubuntu1604_bdw/software/Core/EasyBuild/3.8.1/lib/python2.7/site-packages/easybuild_framework-3.8.1-py2.7.egg/easybuild/framework/easyconfig/easyconfig.py", line 1157, in generate_template_values
    new_val = curr_val % self.template_values
ValueError: incomplete format
```

Reply:
I have:
```
dependencies = [
    ('zlib', '1.2.11'),
    ('Perl', '5.28.0'),
    ('Clang', '6.0.1'),
    ('Exporter-Lite', '0.08', '-Perl-%(perlver)s'),
    ('Getopt-Tabular', '0.3', '-Perl-%(perlver)s'),
]
```